src/api.h

```diff
@@ -36,6 +44,26 @@ AWS_CRT_API const char *aws_crt_error_name(int);
 AWS_CRT_API const char *aws_crt_error_debug_str(int);
 AWS_CRT_API void aws_crt_reset_error(void);
 
+void *aws_crt_current_thread_id(void);
```

Comment:
What purpose does this serve?  Are we expecting people to do platform-specific handling?

Reply:
See https://github.com/awslabs/aws-crt-php/pull/24

Reply:
This will go away, don't need it